include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -1146,6 +1130,26 @@ struct OrtApi {
   */
   ORT_API2_STATUS(ModelMetadataGetGraphDescription, _In_ const OrtModelMetadata* model_metadata,
                   _Inout_ OrtAllocator* allocator, _Outptr_ char** value);
+
+  /**
+  * Use this API to create the configuration of a CUDA Execution Provider
+  */
+  ORT_API2_STATUS(CreateCUDAProviderOptions, _Outptr_ OrtCUDAProviderOptions** out);
+
+  /**
+  * Use this API to set the appropriate configuration knobs of a CUDA Execution Provider
```

Comment:
>se this API to set the ap [](http://example.com/codeflow?start=5&length=25)

The encoding must be mentioned. Also we should always document Doxygen params, although reference to external doc is also helpful.